templates/base/packages/nextjs/hooks/scaffold-eth/useScaffoldEventHistory.ts

```diff
@@ -56,95 +56,91 @@ export const useScaffoldEventHistory = <
     chainId: targetNetwork.id,
   });
 
-  const readEvents = async (fromBlock?: bigint) => {
-    setIsLoading(true);
-    try {
-      if (!deployedContractData) {
-        throw new Error("Contract not found");
-      }
+  const readEvents = useCallback(
+    async () => {
+      setIsLoading(true);
+      try {
+        if (!deployedContractData) {
+          throw new Error("Contract not found");
+        }
 
-      if (!enabled) {
-        throw new Error("Hook disabled");
-      }
+        if (!enabled) {
+          throw new Error("Hook disabled");
+        }
```

Comment:
Umm, I think because in https://github.com/scaffold-eth/scaffold-eth-2/pull/827 we wrapped this function inside `useCallback` which made it indent a bit more ?

Reply:
But doesn't happen to other wrapped stuff

![image](https://github.com/scaffold-eth/create-eth/assets/2486142/acbdb8c2-90f8-4533-9186-4aed8050b456)

:man_shrugging:

Reply:
Ohh yeah :(, tried asking chatGPT and it does says there might be some extra indentation added or might be some problem with diffing UI. 

But I just checked out the commit before https://github.com/scaffold-eth/scaffold-eth-2/pull/827 and indentation was looking good there too : 

![Screenshot 2024-05-01 at 1 53 49 PM](https://github.com/scaffold-eth/create-eth/assets/80153681/c6b06d6b-905d-47ed-87c9-6925950a615a)

Reply:
Yeah, probably the UI diff problem. I was wondering if it was caused by having the "wrong space" (non-printable character) or something, even if it looks good visually. 

Anyway, I was just curious about this  haha